From e878faa53c6d7bd8fc3b061a3ae95c24a1120a92 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Mon, 28 Nov 2022 12:29:22 +0200
Subject: [PATCH] [SW-111440] greco: update pcie phy configuration after characterisation

Update PCIe preset vector and enabling loopback entrance.

This change purpose is to optimize PCI Gen4 link.

Change-Id: Iff43043fd004d21a8fc294a3bd2c75459cf2179f
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/arch/greco/common/common.c b/arch/greco/common/common.c
index 2a537b3..5052210 100644
--- a/arch/greco/common/common.c
+++ b/arch/greco/common/common.c
@@ -568,11 +568,11 @@
 
 	/* Disable the EQ REDO Gen3 */
 	regmap_writel(0x00402801, mmPCIE_DBI_GEN3_RELATED_OFF);
-	regmap_writel(0x04027041, mmPCIE_DBI_GEN3_EQ_CONTROL_OFF);
+	regmap_writel(0x05027071, mmPCIE_DBI_GEN3_EQ_CONTROL_OFF);
 
 	/* Disable the EQ REDO Gen4 */
 	regmap_writel(0x01402801, mmPCIE_DBI_GEN3_RELATED_OFF);
-	regmap_writel(0x04027041, mmPCIE_DBI_GEN3_EQ_CONTROL_OFF);
+	regmap_writel(0x05027071, mmPCIE_DBI_GEN3_EQ_CONTROL_OFF);
 
 	start = timestamp_get_us();
 
